R/spark_shell.R

```diff
@@ -7,6 +7,68 @@ shell_connection_validate_config <- function(config) {
   config
 }
 
+databricks_connection <- function(config, extensions) {
+  tryCatch({
+    gatewayPort <- as.numeric(SparkR:::callJStatic("com.databricks.backend.daemon.driver.RDriverLocal",
```

Comment:
You need to call the sparklyr equivalent here, see http://spark.rstudio.com/reference/sparklyr/latest/invoke.html